extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -72,35 +66,35 @@ public function executeIndex() {
 		wfProfileOut( __METHOD__ );
 	}
 
-	public function executeThemeTab() {
+	public function themeTab() {
 
 	}
 
-	public function executeCustomizeTab() {
+	public function customizeTab() {
 
 	}
 
-	public function executeWordmarkTab() {
-		$this->faviconUrl = Wikia::getFaviconFullUrl();
+	public function wordmarkTab() {
+		$this->response->setData( [
+			'faviconUrl' => Wikia::getFaviconFullUrl(),
+			'token' => $this->wg->User->getEditToken(),
+		] );
 	}
 
-	public function executePicker() {
-
+	public function picker() {
+		// SUS-797: Add edit token for background image upload
+		$this->response->setVal( 'token', $this->wg->User->getEditToken() );
 	}
 
-	public function executePreview() {
-		global $wgEnablePortableInfoboxEuropaTheme;
-
-		$this->infoboxTheme = $wgEnablePortableInfoboxEuropaTheme ? 'theme="europa"' : '';
+	public function preview() {
+		$this->infoboxTheme = $this->wg->EnablePortableInfoboxEuropaTheme ? 'theme="europa"' : '';
 	}
 
 	/**
 	 *
 	 * @author Federico "Lox" Lucignano
 	 */
 	private function getUploadErrorMessage( $status ) {
-		global $wgFileExtensions, $wgLang;
-
 		switch( $status[ 'status' ] ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "status" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)